otcextensions/sdk/vpc/v1/_proxy.py

```diff
@@ -21,6 +22,65 @@
 class Proxy(proxy.Proxy):
     skip_discovery = True
 
+    # ======== Bandwidth ========
+    def assign_bandwidth(self, **attrs):
+        """Assign bandwidth
+
+        :param dict attrs: Keyword arguments which will be used to assign
+            a :class:`~otcextensions.sdk.vpc.v1.bandwidth.Bandwidth`
+        """
+        project_id = self.get_project_id()
+        return self._create(_bandwidth.Bandwidth, project_id=project_id,
+                            **attrs)
+
+    def add_eip_to_bandwidth(self, bandwidth, publicip_info):
```

Comment:
please add more detailed info in doc about publicip_info structure

Reply:
I would say include this into docs as such